src/main/webapp/app/exam/manage/suspicious-behavior/suspicious-behavior.component.html

```diff
@@ -0,0 +1,31 @@
+<h5>{{ 'artemisApp.examManagement.suspiciousBehavior.title' | artemisTranslate }}</h5>
+
+<p>
+    {{ 'artemisApp.examManagement.suspiciousBehavior.suspiciousSessionDescription' | artemisTranslate }}
+</p>
+
+<ul>
+    <li>
+        {{ 'artemisApp.examManagement.suspiciousBehavior.suspiciousSessionCriterionIpAddress' | artemisTranslate }}
+    </li>
+    <li>
+        {{ 'artemisApp.examManagement.suspiciousBehavior.suspiciousSessionCriterionBrowserFingerprint' | artemisTranslate }}
+    </li>
+</ul>
+
+<p>
+    {{ 'artemisApp.examManagement.suspiciousBehavior.suspiciousSessions.number' | artemisTranslate : { numberOfSuspiciousSessions: suspiciousSessions.length } }}
+</p>
+
+<button *ngIf="suspiciousSessions.length !== 0" [disabled]="suspiciousSessions.length === 0" type="submit" (click)="goToSuspiciousSessions()" class="btn btn-primary mb-5">
```

Comment:
```suggestion
<button *ngIf="suspiciousSessions.length" [disabled]="suspiciousSessions.length === 0" type="submit" (click)="goToSuspiciousSessions()" class="btn btn-primary mb-5">
```
0 is falsy

Reply:
And you only want to display it when there are suspiciousSessions, BUT: you disable it when there are no sessions? It's not rendered anyway, why would you disable it?

Reply:
this is a leftover because initially the button was only disabled and not hidden. Removed the disabled directive now.